src/librustc_borrowck/borrowck/mir/elaborate_drops.rs

```diff
@@ -586,6 +586,11 @@ impl<'b, 'tcx> ElaborateDropsCtxt<'b, 'tcx> {
                             // drop elaboration should handle that by itself
                             continue
                         }
+                        TerminatorKind::Resume => {
+                            // We can replace resumes with gotos
+                            // jumping to a canonical resume.
```

Comment:
Hmm, in an earlier version of this PR I definitely needed to address Resume in this loop, because otherwise it falls into that `patch.is_patched` assertion below. But I'm guessing that this was an artifact of the dataflow changes that have since been moved to a different PR. I'll take this out and save figuring out whether its still necessary for later.